src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.scss

```diff
@@ -55,23 +55,14 @@ $box-height: 206px;
 }
 
 .box__add-row-class-button {
+    align-items: center;
+    border: 0;
     cursor: pointer;
-    position: absolute;
-    background-color: $brand-secondary;
-    z-index: 1;
-    left: -14px;
-    width: 16px;
+    display: flex;
     height: $box-height;
-    border: 0;
     padding: 0;
-
-    span {
-        white-space: nowrap;
-        text-transform: uppercase;
-        color: $white;
-        display: block;
-        writing-mode: vertical-lr;
-        position: relative;
-        transform: rotate(180deg);
-    }
+    position: absolute;
+    right: -23px;
+    width: 32px;
```

Comment:
$basic-padding ?

Reply:
The button that is inside this div is 32px, were matching that width, we don't want button sizes to be related to paddings.